models/UserSearch.php

```diff
@@ -95,6 +99,7 @@ public function search($params)
 
         $query->andFilterWhere(['like', 'username', $this->username])
             ->andFilterWhere(['like', 'email', $this->email])
+            ->andFilterWhere(['user.id' => $this->id])
```

Comment:
In fact, the correct one would be to use `$query->modelClass::tableName() . '.id'`

We should always remember that models can be remapped.

Please make the change and I will accept the PR.